draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,15 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint SHOULD limit the state it commits to retiring connection IDs using
+RETIRE_CONNECTION_ID by limiting the number of connection IDs that it tracks for
+retirement to at least twice the active_connection_id_limit. An endpoint MUST
+NOT forget a connection ID without retiring it, though MAY choose to treat
+having too many connection IDs in need of retirement as a connection error of
```

Comment:
"too many" seems non-specific here. I think we need more detail.

Reply:
How about "though it MAY choose to treat having more connection IDs in need of retirement than is consistent with a manageable amount of states as a connection error..."?

A number doesn't feel right here, but I'm open to wording suggestions.

Reply:
Instead of "too many" how about "having connection IDs in need of retirement that exceed this limit..."  or "in excess of this limit". That way we don't have ambiguity about the referent of "too many"

Reply:
done